src/tests/rules/require-author.test.ts

```diff
@@ -0,0 +1,35 @@
+import { rules } from "../../rules/require-properties.js";
+import { ruleTester } from "./ruleTester.js";
+
+ruleTester.run("require-author", rules["require-author"], {
+	invalid: [
+		{
+			code: "{}",
+			errors: [
+				{
+					data: { property: "author" },
+					line: 1,
+					messageId: "missing",
+				},
+			],
+		},
+		{
+			code: `{
+            "name": "foo",
+            "version": "1.0.0"
+            }
+            `,
+			errors: [
+				{
+					data: { property: "author" },
+					line: 1,
+					messageId: "missing",
+				},
+			],
+		},
+	],
+	valid: [
+		`{ "main": "./index.js", "author": "Sophie Trudeau" }`,
+		`{ "author": "Jessica Moss" }`,
```

Comment:
[Question] What is the reference with these names? Google isn't helping me 😄

Reply:
They're both members of an amazing post-rock band, Thee Silver Mt. Zion,  which spun out of pioneer post-rock band Godspeed You! Black Emperor.  Sophie's also a member of GYBE, and Jessica is married to Efrim who's also in GYBE.  I originally had two other people, but cspell didn't recognize the words "Reznor", "Efrim", or "Menuck", so I went with these 😊